packages/lib-user/src/components/MyGroups/components/GroupCard/GroupCard.js

```diff
@@ -0,0 +1,110 @@
+import { SpacedHeading } from '@zooniverse/react-components'
+import { Anchor, Box } from 'grommet'
+import { number, string } from 'prop-types'
+import styled, { css } from 'styled-components'
+
+import { TitledStat } from '@components/shared'
+
+const StyledListItem = styled.li`
+  border-radius: 8px;
+  list-style: none;
+  padding: 20px;
+  
+  &:hover, &:focus-within {
+    box-shadow: 1px 2px 6px 0px rgba(0, 0, 0, 0.25);
+  }
+`
+
+const StyledAnchor = styled(Anchor)`
+  &:hover, &:focus {
+    text-decoration: none;
+    outline: none;
+    box-shadow: none;
+  }
+`
+
+const StyledRole = styled(Box)`
+  font-size: 9px;
+  letter-spacing: 0.9px;
+  display: flex;
+  align-items: center;
+  width: 90px;
+  line-height: normal;
+  padding: 5px 0;
+  text-transform: uppercase;
+  font-weight: bold;
+  ${props =>
+    css`
+      color: ${props.theme.global.colors.black};
+    `}
+`
+
+function GroupCard({
+  classifications = 0,
+  contributors = 0, // members
+  displayName = '',
+  hours = 0,
+  id = '',
+  projects = 0,
+  role = ''
+}) {
+  return (
+    <StyledListItem>
+      <StyledAnchor
+        href={`https://local.zooniverse.org:8080/?groups=${id}`}
```

Comment:
The `StyledAnchor` component will change per a subsequent PR that incorporates next/link and internal routing.